build.gradle

```diff
@@ -119,6 +119,9 @@ allprojects {
   repositories {
     mavenCentral()
     mavenLocal()
+    maven {
+      url = uri("https://repository.apache.org/content/repositories/orgapacheparquet-1065")
```

Comment:
Just a note: this is temporary during the Parquet release vote (just to not forget to remove this once Parquet release is out 😄 )